test/dotnet.Tests/CompletionTests/ShellDiscoveryTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable disable
+
+
+using System.CommandLine.StaticCompletions.Shells;
+using Microsoft.DotNet.Cli.Commands.BuildServer.Shutdown;
+using Xunit;
+using Xunit.Abstractions;
+using Parser = Microsoft.DotNet.Cli.Parser;
+
+namespace System.CommandLine.StaticCompletions.Tests;
+
+public class ShellDiscoveryTests()
+{
+    [Fact]
```

Comment:
maybe parameterize this test with the shell names and set the SHELL env var before each run so we can easily test all of our shell experiences??